SergiyGoryainov/HW06/HW06-1.py

```diff
@@ -0,0 +1,13 @@
+even = []
+odd3 = []
+other = []
+
+for i in range(1, 11):
+    even.append(i) if i % 2 == 0 else \
+        odd3.append(i) if i % 3 == 0 \
+            else other.append(i)
```

Comment:
even = []
odd = []
other = []

for number in range(1, 11):
    if number >= 2 and number % 2 == 0:
        even.append(number)
    elif number >= 3 and number % 3 == 0:
        odd.append(number)
    else:
        other.append(number)

print(f"Even numbers that are divisible by 2: {even}")
print(f"\nOdd numbers, which are divisible by 3: {odd}")
print(f"\nNumbers that are not divisible by 2 and 3: {other}")